src/lib/transaction_logic/mina_transaction_logic.ml

```diff
@@ -20,6 +20,31 @@ module type S = sig
 
   type location
 
+  module Inputs :
```

Comment:
In general we should make it clear what's for Zeko. It would also help if you added a comment here.